src/xcode/ENA/ENA/Source/Scenes/Onboarding/OnboardingInfoViewController.swift

```diff
@@ -149,6 +158,28 @@ final class OnboardingInfoViewController: UIViewController {
 		present(alert, animated: true, completion: nil)
 	}
 
+	private func loadCountryList() {
+		client?.supportedCountries(completion: { [weak self] result in
+			var availableCountries: [Country]
+			switch result {
+			case .failure:
+				logError(message: "Did not receive a country list.")
+				availableCountries = []
+			case .success(let countries):
+				availableCountries = countries
+			}
+
+			self?.supportedCountries = availableCountries
+				.sorted { $0.localizedName.localizedCompare($1.localizedName) == .orderedAscending }
+		})
+	}
+
+	private func updateCountrySection(countries: [Country]) {
+		let bla = UIView(frame: CGRect(x: 0, y: 0, width: 200, height: 50))
```

Comment:
Blub? :D

Reply:
oh my god, leftover code i overlooked. Thanks so much for pointing it out!!!